content/commands/ts.add/index.md

```diff
@@ -194,7 +219,7 @@ Use it only if you are creating a new time series. It is ignored if you are addi
 
 Returns one of these replies:
 
-- [Integer reply]({{< relref "/develop/reference/protocol-spec#integers" >}}) - the timestamp of the upserted sample
+- [Integer reply]({{< relref "/develop/reference/protocol-spec#integers" >}}) - the timestamp of the upserted sample. For an element that is ignored (see the `IGNORE` parameter above), the value will be `max_timestamp`.
```

Comment:
We should explain `max_timestamp` (it is the timestamp of the sample with the largest timestamp in the time series). Also `value_at_max_timestamp` (the value at that `max_timestamp`).